dotcom-rendering/scripts/webpack/server.js

```diff
@@ -19,6 +32,8 @@ module.exports = ({ sessionId }) => ({
 	},
 	externals: [
 		'@loadable/component',
+		'express',
+		'log4js',
```

Comment:
Express & log4js both use expressions in `require()`

This is where require takes an expression, e.g
```js
function getModule (name) {
    return require(name)
}```

Which is not suited to webpack ~ as it is unable to know what it is supposed to bundle to meet the requirements of the module. By specifying these are external to the build, they never get bundled. 

I'll be testing this on CODE to make sure everything works as expected.

Reply:
It would be really useful to have this in the code for future editors as we don't have a test explaining it.